src/components/common/utils.spec.ts

```diff
@@ -76,18 +90,88 @@ export class FormAssociatedTestBed<T extends IgcFormControl> {
       },
       { once: true }
     );
+
     this.form.requestSubmit();
     return data;
   }
 
-  public submitValidates(msg?: string) {
-    expect(this.submit(), msg).not.to.be.undefined;
-    expect(this.valid).to.be.true;
+  /**
+   * Assigns properties to the element and optionally waits for the update.
+   *
+   * This function takes an object of properties to assign to the element.
+   * It then uses `Object.assign` to merge these properties into the element's properties.
+   * If the `waitForUpdate` parameter is `true`, the function waits for the element to be updated before returning.
+   */
+  public async setProperties(
+    props: { [K in keyof T]?: T[K] | string },
+    waitForUpdate = true
+  ): Promise<void> {
```

Comment:
Can this be switched to false since the only test awaiting is in the mask that can request it and it's not needed for the rest